packages/fluentui/react-northstar/test/specs/components/Tree/Tree-test.tsx

```diff
@@ -204,5 +213,19 @@ describe('Tree', () => {
         expect.objectContaining({ activeItemIds: expect.arrayContaining(['2', '21', '1']) }),
       );
     });
+
+    it("should contain index of item open at clicking the title's children", () => {
+      const wrapper = mountWithProvider(<Tree items={items} />);
+
+      getTitles(wrapper)
+        .at(1) // title '2'
+        .simulate('click');
+      checkOpenTitles(wrapper, ['1', '2', '21', '22', '3']);
```

Comment:
This assertion shouldn't be necessary

Reply:
Not a big deal for me but - I'm not a fan of assertions that are not related to the test - better to only test one specific thing, you should already have another test covering this assertion.

I generally follow this pattern http://wiki.c2.com/?ArrangeActAssert

It's just so easy to know what a test is supposed to do and find possible wrong causes during maintenance.